docs/time/index.rst

```diff
@@ -1212,6 +1212,16 @@ jd         :class:`~astropy.time.TimeDeltaJD`
 datetime   :class:`~astropy.time.TimeDeltaDatetime`
 =========  ===================================================
 
+.. warning:: The ``datetime`` format does *not* support instants of time that
+             are leap seconds. This is a 
```

Comment:
Getting finicky about type checking, but I think a leap second is an interval of time which is different from an instant of time. Maybe just `.. does *not* support times that are within a leap second.`